src/sdk-wrapper/sdkWrapper.js

```diff
@@ -293,7 +295,6 @@ class OpenTokSDK {
    * @returns {Promise} <resolve: empty, reject: Error>
    */
   connect(eventListeners) {
-    this.off();
```

Comment:
@adrice727 I took this off as it was removing all the listeners from `setInternalListeners`. Not 100% what other effects it could cause. Also it seems anyone could call these and remove the listeners, leaving the connected state unchanged.